tree/ntupleutil/v7/test/ntuple_importer.cxx

```diff
@@ -330,7 +333,11 @@ TEST(RNTupleImporter, STL)
       std::unique_ptr<TFile> file(TFile::Open(fileGuard.GetPath().c_str(), "RECREATE"));
       auto tree = std::make_unique<TTree>("tree", "");
       auto vec = new std::vector<float>{1.0, 2.0};
+      auto pair = new std::pair<float, float>{3.0, 4.0};
+      auto tuple = new std::tuple<int, float, bool>{5, 6.0, true};
```

Comment:
Unfortunately, `delete` is not called on these two.  For `vec`, `pair`, and `tuple`, I would use a `unique_ptr` instead.